charts/images.yaml

```diff
@@ -75,7 +75,7 @@ images:
 - name: cluster-autoscaler
   sourceRepository: github.com/gardener/autoscaler
   repository: eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler
-  tag: "v0.15.0"
+  tag: "v0.16.0"
```

Comment:
Why not `v1.16.1`?

```suggestion
  tag: "v0.16.1"
```

Reply:
Nice the changes on master are due to two different PRs I thought to raise them as two different ones. However, yes I can club the changes into one.

Reply:
Could you please transport the release notes for `cluster-autoscaler@v1.16.1` as well?
Could you please file cherry-pick to `release-v1.20` as well?

Reply:
>Could you please transport the release notes for cluster-autoscaler@v1.16.1 as well?
Done. 

> Could you please file cherry-pick to release-v1.20 as well?
Yes, will raise it next.

Reply:
This comment is still not resolved.